frontend/public/components/namespace.jsx

```diff
@@ -230,12 +230,15 @@ class NamespaceDropdown_ extends React.Component {
     const model = getModel(useProjects);
     const allNamespacesTitle = `all ${model.labelPlural.toLowerCase()}`;
     const items = {};
+
+    const sortedData = _.map(data, 'metadata.name').sort();
```

Comment:
I'd call this `sortedNames` since it's just the names, not the data.

Reply:
True, will do.